Tools/PostProcessing/RunsScan/BeamPropToHDF5.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/env python3
+
+###
+### Python script to evaluate lab frame particle data
+### and save the beam (RMS) properties in an .h5 file
+###
+### L.D.Amorim @ LBNL
+### 3 Sep 2020
+###
```

Comment:
```suggestion
### License: BSD-3-Clause-LBNL
###
```

Reply:
@ax3l , oh sorry for missing this!
I will fix all of these in a commit.
Thanks for catching